lib/__tests__/findCurrentImports-test.js

```diff
@@ -0,0 +1,30 @@
+import Configuration from '../Configuration';
+import findCurrentImports from '../findCurrentImports';
+import parse from '../parse';
+
+it('finds imports', () => {
+  const currentFileContent = `
+const React = require('react');
```

Comment:
Good call. 079aeec

Most of this is covered through the Importer test, that's why it's not very fleshed out.